jihyeon/app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const cors = require('cors');
+const morgan = require ("morgan");
+const http = require ("http");
+require("dotenv").config();
+
```

Comment:
```suggestion
require('dotenv').config();

const logger = require('morgan');
const express = require('express');
const cors = require('cors');
const { DataSource } = require('typeorm');

```

- 위 내용과 같이 환경변수를 호출하는 config 문을 제일 위에서 선언해주세요!
- 코드 시작과 동시에 필요한 환경변수 세팅을 확실히 하기 위한 컨벤션이라고 생각해주시면 되겠습니다!